server/mailet/integration-testing/src/test/java/org/apache/james/mailets/ForwardLoopIntegrationTest.java

```diff
@@ -131,7 +132,7 @@ void tearDown() {
 
     @Test
     void filterForwardShouldNotCreateLoopError() throws Exception {
-        filteringManagementProbe.defineRulesForUser(ALICE, asRule(Forward.to(BOB.asMailAddress()).keepACopy()));
+        filteringManagementProbe.defineRulesForUser(ALICE, asRule(Forward.to(BOB.asMailAddress()).withoutACopy()));
```

Comment:
Add maybe an other test, where you keep a copy, and you have a mail stored for alice and cedric :)

Reply:
added